system/ee/installer/updates/ud_3_05_05.php

```diff
@@ -49,7 +49,7 @@ public function do_update()
      */
     private function normalizeFieldLayoutData()
     {
-        $layouts = ee('Model')->get('ChannelLayout')->all();
+        $layouts = ee('Model')->get('ChannelLayout')->fields('field_layout')->all();
```

Comment:
I assume this is also unrelated to this change? @intoeetive

Reply:
Correct